pkg/target/target.go

```diff
@@ -595,73 +605,31 @@ func tplFuncMap() template.FuncMap {
 	return f
 }
 
-func processTemplateValues(valuesMap map[string]interface{}, templateContext map[string]interface{}) (map[string]interface{}, error) {
-	tplFn := template.New("values").Funcs(tplFuncMap()).Option("missingkey=error")
-	recursionDepth := 0
-	tplResult, err := templateSubstitutions(valuesMap, templateContext, tplFn, recursionDepth)
+func processTemplateValues(helmValues map[string]interface{}, templateContext map[string]interface{}) (map[string]interface{}, error) {
+	data, err := kyaml.Marshal(helmValues)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to unmarshal helm values template: %w", err)
```

Comment:
The error says "unmarshal" but the function the error comes from is `Marshal`. Should the error message say unmarshal?